server/src/main/java/org/elasticsearch/action/admin/cluster/remote/RemoteClusterNodesAction.java

```diff
@@ -43,13 +44,25 @@ public RemoteClusterNodesAction() {
     }
 
     public static class Request extends ActionRequest {
+        private final boolean remoteClusterServer;
 
-        public static final Request INSTANCE = new Request();
+        public Request() {
+            this.remoteClusterServer = false;
```

Comment:
I chose to default retrieve all nodes instead of server nodes since the action class is called RemoteClusterNodesAction instead of RemoteClusterServerNodesAction.